eureka/src/main/java/com/linecorp/armeria/server/eureka/EurekaUpdatingListener.java

```diff
@@ -152,10 +152,11 @@ public static EurekaUpdatingListenerBuilder builder(
     @Override
     public void serverStarted(Server server) throws Exception {
         final InstanceInfo newInfo = fillAndCreateNewInfo(instanceInfo, server);
```

Comment:
nit: How about directly assigning to `instanceInfo`?

Reply:
Ok, changed.